cmd/webhook/main.go

```diff
@@ -28,13 +26,11 @@ func main() {
 	stopCh, exit := util.SetupExitHandler(util.GracefulShutdown)
 	defer exit() // This function might call os.Exit, so defer last
 
-	logf.InitLogs(flag.CommandLine)
+	logf.InitLogs()
 	defer logf.FlushLogs()
 
 	cmd := app.NewServerCommand(stopCh)
-	cmd.Flags().AddGoFlagSet(flag.CommandLine)
 
-	flag.CommandLine.Parse([]string{})
```

Comment:
I am not sure what this line was doing before. I imagine there is no impact removing it (I hope)